app/acknowledgement/process-invalid.js

```diff
@@ -3,14 +3,20 @@ const { getHoldCategoryId } = require('../holds')
 const getHoldCategoryName = require('./get-hold-category-name')
 const holdAndReschedule = require('../reschedule')
 const { resetPaymentRequestById } = require('../reset')
+const { sendAckowledgementErrorEvent } = require('../event')
 
-const processInvalid = async (schemeId, paymentRequestId, frn, message) => {
+const config = require('../config')
+
+const processInvalid = async (schemeId, paymentRequestId, frn, acknowledgement) => {
   const transaction = await db.sequelize.transaction()
   try {
     await resetPaymentRequestById(paymentRequestId, schemeId, transaction)
-    const holdCategoryName = getHoldCategoryName(message)
+    const holdCategoryName = getHoldCategoryName(acknowledgement.message)
```

Comment:
This function uses our new `getHoldCategoryName`

Reply:
I'm not sure what you mean here, getHoldCategoryName is a function in itself that decides which category to use (Dax rejection or Bank account anomaly) when message is 'Invalid bank details'. Do you mean inside the function itself that it uses the new constants?